runtime/doc/options.txt

```diff
@@ -2857,10 +2750,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 					*'fileformats'* *'ffs'*
 'fileformats' 'ffs'	string (default:
-				Vim+Vi	MS-DOS and MS-Windows: "dos,unix",
+				Vim+Vi	Win32: "dos,unix",
 				Vim	Unix: "unix,dos",
 				Vim	Mac: "mac,unix,dos",
-				Vi	Cygwin: "unix,dos",
```

Comment:
IIRC we aren't supporting Cygwin. I assume you know that, just want to make my intention clear.